### PR TITLE
Add the static-roles feature for MSSQL

### PR DESCRIPTION
**Pull request by johnathanschmidt**
* Implement SetCredentials function with proper SQL statement
* Implement prepareMSSQLTestContainer to create test container
* Refactor existing tests to use the container
- Inspired by the PostgreSQL and MySQL plugin implementations

### CODE REVIEW

**Comment by johnathanschmidt** [comment]
To resolve #8930

**Comment by Valarissa** [comment]
Just an FYI, I just merged in a change to add `helper/testhelpers/mssql/mssqlhelper.go` and a PrepareMSSQLTestContainer as part of that package.Thank you for doing this though!

**Comment by pbernal** [comment]
Hi @johnathanschmidt - if you would like to continue on seeing this through, could you please sign the CLA? Once you sign the CLA, we will do our best to try to get some 👀 on it for a review.

**Comment by johnathanschmidt** [comment]
Hi @pbernal , I have signed the CLA, but I think that was the wrong way. I put my company e-mail, not my github account e-mail, it's possible to change after signed ?

**Comment by kalafut** [comment]
@john-osullivan I believe the CLA check passes if the email you created the commit with is in your github settings. If you update Github to include this email (you can have multiple emails), the check will pass without having to repush anything.

**Comment by johnathanschmidt** [comment]
Hi @kalafut , thanks for the tip!

@pbernal , now it's ok!

**Comment by austingebauer** [comment]
Hi, @johnathanschmidt. I'm happy to review this PR and help you get this into Vault. Once you've updated your branch by pulling in the latest changes from master, I'll start having a look. Thanks!

**Comment by johnathanschmidt** [comment]
Hi @austingebauer, I updated my branch with the master branch changes, if there is anything I can do

**Comment by austingebauer** [comment]
Thanks, @johnathanschmidt. I'm reviewing and testing this out now.

File: plugins/database/mssql/mssql_test.go [hunk 1]
```diff
@@ -123,6 +124,136 @@ func TestMSSQL_RotateRootCredentials(t *testing.T) {
...
+	}
+}
+
+func TestMSSQL_SetCredentials(t *testing.T) {
+	type testCase struct {
+		rotationStmts []string
+	}
+
+	tests := map[string]testCase{
+		"username rotation": {
```
[the remark this answers is not in the document]
Comment (johnathanschmidt) [reply plugins/database/mssql/mssql_test.go]: Nice! I made this change

**Pull request merged**